7.1.11-fpm-alpine3.4/Dockerfile

```diff
@@ -48,6 +48,11 @@ ENV MAILHOG_VERSION v0.1.9
 RUN curl -fSL "https://github.com/mailhog/MailHog/releases/download/${MAILHOG_VERSION}/MailHog_linux_amd64" -o /usr/local/bin/mailhog \
     && chmod +x /usr/local/bin/mailhog
 
+# Install ldap module.
+RUN apk add --no-cache ldb-dev libldap openldap-dev
+RUN docker-php-ext-configure ldap
+RUN docker-php-ext-install ldap
+
```

Comment:
I think we can just reuse what we are doing here https://github.com/sparkfabrik/docker-php-base-image/blob/94d2845c8af49a50677b8c8a69c2520eededd8d1/7.1.11-fpm-alpine3.4/Dockerfile#L29 and add `openldap-dev` to PHP_EXTRA_DEPS line 25.

Another thing that is missing is the `ldap.ini` file, it should part of this repo.

I would also add a test to check that is enabled/disabled.

Reply:
Ok great! The file `ldap.ini` has been added to `config.disabled` folder in each image folder.